config/identity/handler/provider-factory/identity.json

```diff
@@ -2,7 +2,7 @@
   "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^1.0.0/components/context.jsonld",
   "@graph": [
     {
-      "comment": "Sets all the relevant oidc parameters.",
+      "comment": "Sets all the relevant oidc parameters. webid claim is in openid scope until an official scope has been decided.",
```

Comment:
Reference to spec issue URL? Bonus points for `seeAlso` 😃

Reply:
Components.js tries to interpret it a bit too much so it'll just have to be a string in the comment.